.github/workflows/build-deb-pkg.sh

```diff
@@ -0,0 +1,32 @@
+PKG_ROOT=/tmp/avalanche
+DEBIAN_BASE_DIR=$PKG_ROOT/debian
+AVALANCHE_BUILD_BIN_DIR=$DEBIAN_BASE_DIR/usr/local/bin
+AVALANCHE_LIB_DIR=$DEBIAN_BASE_DIR/usr/local/lib/avalanche
+TEMPLATE=.github/workflows/debian/template 
+DEBIAN_CONF=$DEBIAN_BASE_DIR/DEBIAN
+
+mkdir -p $PKG_ROOT/$DEBIAN_BASE_DIR
```

Comment:
Won't this create `/tmp/avalanche/tmp/avalanche/debian`? $DEBIAN_BASE_DIR already includes $PKG_ROOT

Reply:
you are right of course